### PR TITLE
Draft/europython

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

File: memestra/talk.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- ``gast`` is used by ``pythran``, ``tangent``, ``beniget``…
+
+- ``beniget`` is used by ``pythran``, ``memestra``, ``transonic``
+
+----
+
+``memestra``
+============
+
+Original goal: detect use of deprecated definitions flagged with a decorator in Python modules
```
Comment (lsix) [inline memestra/talk.rst:53]: I know you “only” talk about static check, but might make sense to mention here how deprecation are “usually” handled in the python world (`DeprecationWarning` and `PendingDeprecationWarning`)

File: memestra/talk.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+Extended goal: applicable to notebooks
+
+----
+
+Basic Example
+=============
+
+.. code-block:: python
+
+    import decorator
```
Comment (lsix) [inline memestra/talk.rst:64]: out of curiosity, where is this one defined?

I am running python-3.8 here, `decorator` can be imported but `@decorator.deprecated` does not seem to exist

File: memestra/talk.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+- Goto declaration (~= def) / usage (~= uses)
+
+----
+
+Going Further
+=============
+
+- Handle IPython dialect (magics, auto variables…)
+
+- Standard deprecation decorator with runtime behavior
```
Comment (lsix) [inline memestra/talk.rst:149]: probably relates to my previous comments (both?)

File: europython2020/talk.rst [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+===================================
+
+.. code-block:: console
+
+    > python -c "import ast; print(ast.dump(ast.parse('a[1, ...]')))"
+    ...
+
+----
+
+In Python2
```
Comment (lsix) [inline europython2020/talk.rst]: Python2.7 probably (later you differenciate 3.6&3.9)

File: europython2020/talk.rst [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+    use without any other intervening definitions
+
+----
+
+Typical Usage
+=============
+
+A def *without* a use means a useless def:
+
+- Detect unused imports
```
Comment (lsix) [inline europython2020/talk.rst:176]: is useless unless you have expected side-effects in your imports (in `__init__.py`)… but I would not recommend relying on this too much ;)

File: europython2020/talk.rst [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+
+
+----
+
+Being Pedantic
+==============
+
+In Python, one does **not** assign a value to a variable,
+
+One sets an **identifier** on a **value**.
```
Comment (lsix) [inline europython2020/talk.rst:189]: plus refcound increment no?

File: europython2020/talk.rst [hunk 7]
```diff
@@ -0,0 +1,374 @@
...
+:slide-numbers: true
+:css: font.css
+
+
+Intro
+=====
+
+**Python** and **Static Analysis**…
+
+Should be Python **or** Static Analysis!
```
Comment (lsix) [inline europython2020/talk.rst]: you should add a  “spoiler alert” ^_^

File: europython2020/talk.rst (hunk 5 above)
Comment (serge-sans-paille) [reply europython2020/talk.rst:176]: Correct! I'll mention int during the talk :-)

Comment (serge-sans-paille) [reply europython2020/talk.rst:176]: Sometimes you try to import a module just to check it's there too

File: europython2020/talk.rst (hunk 6 above)
Comment (serge-sans-paille) [reply europython2020/talk.rst:189]: well, refcount is a side effect of the binding

File: memestra/talk.rst (hunk 2 above)
Comment (marimeireles) [reply memestra/talk.rst:64]: I don't think these codes ever ran.

**Pull request merged**